client/homebrew/utils/versionHistory.js

```diff
@@ -0,0 +1,121 @@
+export const HISTORY_PREFIX = 'HOMEBREWERY-HISTORY';
+export const HISTORY_SLOTS = 5;
+
+// History values in minutes
+const DEFAULT_HISTORY_SAVE_DELAYS = {
+	'0' : 0,
+	'1' : 2,
+	'2' : 10,
+	'3' : 60,
+	'4' : 12 * 60,
+	'5' : 2 * 24 * 60
+};
+
+const DEFAULT_GARBAGE_COLLECT_DELAY = 28 * 24 * 60;
+
+const DEFAULT_STORED_BREW = {
+	shareId  : 'default_brew',
+	expireAt : '2000-01-01T00:00:00.000Z'
+};
```

Comment:
Is DEFAULT_STORED_BREW used anywhere now with your logic change?

Reply:
No, it is not - I thought about using `DEFAULT_STORED_BREW.expireAt` in the last update to `getVersionBySlot()` but immediately realized that it would always be a static value, so I could just set it inside the function.
I have removed `DEFAULT_STORED_BREW` in the latest commit, I'll push it shortly.